test/integration/authorize/limit_headers_test.rb

```diff
@@ -71,6 +71,36 @@ def setup
                  last_response.header['3scale-limit-reset']
   end
 
+  test 'response headers include correct information when rate-limited' do
+    day_limit = { service_id: @service.id,
+                  plan_id: @plan_id,
+                  metric_id: @metric_id,
+                  day: 100 }
+    UsageLimit.save(day_limit)
+
+    hour_limit = { service_id: @service.id,
+                   plan_id: @plan_id,
+                   metric_id: @metric_id,
+                   hour: 10 } # Stricter limit for the hour
+    UsageLimit.save(hour_limit)
+
+    # Go over limits for the hour
+    current_time = Time.now.utc
```

Comment:
Isn't better to do this with explicit time? That way it would be less dynamic and more obvious what the actual numbers are.

Reply:
I agree. It's a bit better because when the tests fail, we'll see a number we can recognize.
I'll change it.